tests/dummy/app/components/demo-page.js

```diff
@@ -0,0 +1,30 @@
+import Component from '@ember/component';
+import $ from 'jquery';
+
+const DEVILS_NUMBER = 666 / 2; // Give this a proper name  (╯°□°）╯︵ ┻━┻
```

Comment:
this is meant to stay this way?

Reply:
I think it is 😂

Reply:
Don't you like easter eggs? :trollface: @telmaantunes

Reply:
I like easter eggs 👀